tests/inputFields.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { test, expect } from '@playwright/test';
+import exp from 'constants';
+
+test.beforeEach(async({page}) => {
+    await page.goto('https://petclinic.bondaracademy.com/welcome')
+})
+
+test.describe('Pet types', () => {
+    test.beforeEach(async({page}) => {
+        await page.getByRole('link', {name: "Pet Types"}).click() //1. Select Pet Types menu
+    })
+
+    test('Update pet type', async({page}) => {
+        const petTypesTableName = page.getByRole('heading', {name: "Pet Types"})
+        await expect(petTypesTableName).toHaveText('Pet Types')  //2. assertion of Pet Types text displayed
```

Comment:
When you want to assert the particular text values on the page, the locator to this web element should not contain the text that you are about to assert. If the text will not be on the page, test will fail because the element not found (waiting for locator....) instead of assertion to be failed.
Update the locator so it will not contain the expected text.

Reply:
For example like this: `page.locator('app-pettype-list h2')`

Reply:
Updated in all three tests. I understand the reason for this.